rspec-core/spec/rspec/core/notifications_spec.rb

```diff
@@ -12,7 +12,7 @@ def self.wrap(text, code_or_symbol)
 
   let(:example) { new_example(:status => :failed) }
   exception_line = __LINE__ + 1
-  let(:exception) { instance_double(Exception, :backtrace => [ "#{__FILE__}:#{exception_line}"], :message => 'Test exception') }
+  let(:exception) { instance_double(Exception, :backtrace => [ "#{__FILE__}:#{exception_line}"], :message => 'Test exception', :detailed_message => 'Test exception') }
```

Comment:
Would this fix the failing specs?

```suggestion
  if RSpec::Support::RubyFeatures.supports_exception_detailed_message?
    let(:exception) { instance_double(Exception, :backtrace => [ "#{__FILE__}:#{exception_line}"], :message => 'Test exception', :detailed_message => 'Test exception') }
  else
    let(:exception) { instance_double(Exception, :backtrace => [ "#{__FILE__}:#{exception_line}"], :message => 'Test exception') }
  end  
```